src/main/java/com/cbs/java_essential/ruslan/homework/lesson8/task2/Main.java

```diff
@@ -0,0 +1,142 @@
+package com.cbs.java_essential.ruslan.homework.lesson8.task2;
+
+import org.apache.log4j.Logger;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.ZonedDateTime;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * by Ruslan svitliy.rk@gmail.com
+ * Homework 8 task2
+ * Требуется: Описать класс с именем Worker, содержащую следующие поля:
+ * фамилия и инициалы работника;
+ * название занимаемой должности;
+ * год поступления на работу.
+ * Написать программу, выполняющую следующие действия:
+ * ввод с клавиатуры данных в массив, состоящий из пяти элементов типа Worker (записи должны быть упорядочены
+ * по алфавиту);
+ * если значение года введено не в соответствующем формате выдает исключение.
+ * вывод на экран фамилии работника, стаж работы которого превышает введенное значение.
+ */
+public class Main {
+    private static final Logger log = Logger.getLogger(Main.class);
+
+    public static void main(String[] args) {
+        Worker workerNew = new Worker();
+        Worker[] workers = new Worker[3];
+
+        List<Worker> listOfWorker;
+        listOfWorker = Arrays.asList(workers);
+
+        BufferedReader reader = new BufferedReader
+                (new InputStreamReader(System.in));
```

Comment:
```suggestion
               BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
```
переместите в одну строку